configs/sites/tier1/aws-pcluster/README.md

```diff
@@ -392,9 +393,9 @@ spack env activate -p envs/unified-env
 #    the line should look like this:
 #      compilers: [%intel']
 
-spack concretize 2>&1 | tee log.concretize.unified-env.001
-./util/show_duplicate_packages.py -d log.concretize.unified-env.001
-spack install --verbose 2>&1 | tee log.install.unified-env.001
+spack concretize 2>&1 | tee log.concretize.001
+${SPACK_STACK_DIR}/util/show_duplicate_packages.py -d log.concretize.001
```

Comment:
I'm curious to learn where `${SPACK_STACK_DIR}` is defined.

Reply:
`SPACK_STACK_DIR` gets set when running `source setup.sh` in the top-level spack-stack directory